src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -224,16 +224,19 @@ class spell_warr_jump_to_skyhold : public SpellScriptLoader
 
             if (Unit* caster = GetCaster())
             {
-                float pos_x = caster->GetPositionX();
-                float pos_y = caster->GetPositionY();
-                float pos_z = caster->GetPositionZ() + 30.0f;
+                if (caster->GetZoneIdFromPosition() == 7886)
```

Comment:
Please add ZONE_DALARAN_LEGION = 7886 in AreaName enum (SharedDefines.h) and use that enum instead of magic number

Reply:
Also, use GetZoneId() instead of GetZoneIdFromPosition() (first one is cached, second one is calculated)